src/main/java/core/basesyntax/AreaCalc.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface AreaCalc {
```

Comment:
The name of the interface 'AreaCalc' should not be a verb. Consider renaming it to something that represents what it is or does, such as 'AreaCalculator', which follows the Java naming convention for interfaces.